WinImageBuilder.psm1

```diff
@@ -848,6 +938,65 @@ function New-MaaSImage {
 }
 
 function New-WindowsOnlineImage {
+    <#
+    .SYNOPSIS
+     This function generates a Windows image using Hyper-V as hypervisor.

```

Comment:
using hyper-v to instantiate the image in order to apply the updates and install cloudbase-init.